src/Components/Server/src/ComponentHub.cs

```diff
@@ -66,6 +66,7 @@ public override Task OnDisconnectedAsync(Exception exception)
             }
 
             CircuitHost = null;
+            circuitHost.UnhandledException -= CircuitHost_UnhandledException;
```

Comment:
This might not make sense, but is there something more general/different we could do inside circuitHost.DisposeAsync to handle this instead of forcing the clients to unregister? Sounds kinda leaky otherwise

Reply:
In fact, could we do `UnhandledException = null;` inside DisposeAsync? https://stackoverflow.com/a/153744

Wouldn't that be cleaner?

Reply:
Or if that doesn't work (which I would expect it to)
Inside dispose?

```csharp
            foreach (var del in UnhandledException.GetInvocationList())
            {
                UnhandledException -= (UnhandledExceptionEventHandler)del;
            }
```

Reply:
What do you think we achieve by doing that? So far you're just proposing an alternative without saying why 😆

Reply:
Discussed offline, I'm going to follow up on the exception handling separately.